### DISCUSSION TITLE
osu!catch editor - Curved sliders should be a thing

### DISCUSSION THREAD

**Discussion by Xinnoh** [ppy/osu#27933] [Ideas] [answered]
Imagine if standard editor could only using bezier curves again, this is what creating sliders in the catch editor feels like.

Common slider is stable vs the same/similar slider in lazer
![screenshot027](https://github.com/ppy/osu/assets/30382015/82b2a6b1-c416-4e30-89de-680a48a70012)

![osu_2024-04-19_17-36-06](https://github.com/ppy/osu/assets/30382015/647c1e96-d500-4988-a628-cee628df20f4)

When mapping, 98% of the sliders I make are either straight sliders, or curved sliders with exactly one slider node. I never use sliders with sharp angles (ie red slider nodes) when I'm mapping in autopilot. Laser can only do sharp angle sliders which interrupts my flow.

In my opinion, basic curved sliders look better than sliders with red slider nodes and can do almost all commonly used patterns.

Technically similar curves can be made by putting a bunch of slider nodes, but it's not easy to quickly modify and requires more effort.

**Comment by Walavouchey** [comment, marked as answer]
the same has been reported in https://github.com/ppy/osu/discussions/25605

please upvote that thread instead
